v3/gssapi/client.go

```diff
@@ -18,7 +22,7 @@ import (
 	"github.com/jcmturner/gokrb5/v8/credentials"
 )
 
-// Client implements ldap.GSSAPIClient interface.
+// Client implements ldapgithub.com/jcmturner/gokrb5/v8/spnego.GSSAPIClient interface.
```

Comment:
This change is only in this (v3/gssapi/client.go) file and not in the gssapi/client.go file.

Additionally it seems to have a typo, probably accidentally introduced.


Until we get our versioning strategy implemented we expect to keep the v3 folder in sync with the root folder (I know, I know)

aside: My recommendation is to make all the changes in only the v3 folder, and then copy the modified files to the root before committing to ensure they're identical.